README.md

```diff
@@ -32,10 +32,10 @@ In addition, EclipseSource also offers [professional support](https://jsonforms.
 
 ### First time setup
 
-- Install [node.js](https://nodejs.org/) (only Node 14 and npm 6 is currently supported)
+- Install [node.js](https://nodejs.org/) (only Node v16.13+ < 17 is currently supported)
+- Install pnpm: <https://pnpm.io/installation>
```

Comment:
Should we specify which version of pnpm or are they pretty compatible?

Reply:
It's hard to say but I'll add a minimum version. This is also specified in the package.json's `engines` prop and will fail when the version constrained is violated.